src/screens/notifications/notifications.tsx

```diff
@@ -70,8 +137,78 @@ export class NotificationsComponent extends React.Component<
         );
     }
 
+    private async markSeen(blockchain: string, id: string) {
+        try {
+            const http = new HttpClient(CONFIG.notificationCenter.markSeenUrl);
+            const res = await http.post('', {
+                notifIds: [id]
+            });
+
+            if (res?.result?.notifications && res.result.notifications) {
+                this.props.markSeenNotification(blockchain, id);
+
+                const { notifications } = this.state;
+                notifications[blockchain][id].seen = true;
+                this.setState({ notifications });
+            }
+        } catch {
+            //
+        }
+    }
+
+    private async fetchNotifications() {
+        const { page, notifications } = this.state;
+
+        try {
+            // Fetch next page
+            const http = new HttpClient(
```

Comment:
same here, let's use actions for this, also by using redux for this we cache the notifications, we minimise the calls to our api. The redux property should not be persisted.